.github/workflows/build-and-test.yaml

```diff
@@ -542,15 +536,16 @@ jobs:
           image_ref=${oci_repo}/cicd/job-image:${image_tag}
 
           echo "oci_image_ref=${image_ref}" >> ${GITHUB_OUTPUT}
```

Comment:
I think we can drop calculation of `oci_repo`, `image_tag` and `image_ref` here as well as setting of `oci_image_ref` output. Instead, the `image-build` job already sets the `image_reference` output which should be equivalent to `oci_image_ref` (has to be also adjusted in the declared outputs of this job of course).

Reply:
good catch. I think  I already switched `image`-step's output accordingly, but missed to clean the - now obsolete - code-snippet calculating image-ref (again)